cpp/util/fake_etcd.cc

```diff
@@ -149,7 +149,7 @@ void FakeEtcdClient::NotifyForPath(const string& path) {
   for (const auto& pair : watches_) {
     if (path.find(pair.first) == 0) {
       for (const auto& cb_cookie : pair.second) {
-        ScheduleCallback(
+        cb_cookie.second->executor()->Add(
```

Comment:
Note to self: these should be flattened, when my other pull request is merged in.